monitoring/psi-prod-cluster/bases/prometheus/prometheus.yaml

```diff
@@ -11,17 +11,14 @@ spec:
       - name: aicoe-sre-alertmanager
         namespace: dh-psi-monitoring
         port: web
-  replicas: 2
```

Comment:
Similar to above, why remove our HA?

Reply:
this namespace has very limited resources, putting these to 3 & 2 respectively would almost max out our cpu and memory

Reply:
Sounds good. I've opened https://github.com/AICoE/idh-manifests/issues/19 so we don't forget. In the meanwhile we can go ahead with the PR